libnd4j/include/ops/declarable/generic/convo/pooling3d_ops.cpp

```diff
@@ -120,37 +120,7 @@ namespace nd4j {
 
             output->assign(0.0);
```

Comment:
Do we still need this assign?

Reply:
indeed, it is unnecessary there, helper does this job by itself